website/docs/r/s3_bucket.html.markdown

```diff
@@ -277,12 +277,22 @@ resource "aws_s3_bucket" "source" {
 
     rules {
       id     = "foobar"
-      prefix = "foo"
       status = "Enabled"
 
+      filter {
+        tags = {}
+      }
```

Comment:
Note: Currently ([on main](https://github.com/hashicorp/terraform-provider-aws/blob/6239d0efdb8a432456bdaa4717fa1697c64ebda9/aws/resource_aws_s3_bucket.go#L2094-L2125)), if there is no `filter` block or it's empty, a default `prefix = ""` is injected. This makes AWS interpret the request as being in schema V1 format (see "Backward compatibility" [here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/replication-add-config.html)). RTC is only enabled on v2, so I updated the syntax here to match v2.